devicetypes/smartthings/zooz-multisiren.src/zooz-multisiren.groovy

```diff
@@ -0,0 +1,198 @@
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ *	Aeon Siren
+ *
+ *	Author: SmartThings
+ *	Date: 2019-03-18
+ */
+ 
+metadata {
+ definition (name: "Zooz Multisiren", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.siren", runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false, vid: "generic-siren-11") {
+	capability "Actuator"
+	capability "Alarm"
+	capability "Switch"
+	capability "Health Check"
+	capability "Temperature Measurement"
+	capability "Relative Humidity Measurement"
+	capability "Battery"
+	capability "Tamper Alert"
+	capability "Refresh"
+
+	fingerprint mfr: "027A", prod: "000C", model: "0003", deviceJoinName: "Zooz S2 Multisiren ZSE19"
+
+ }
+
+ tiles(scale: 2) {
+	multiAttributeTile(name:"alarm", type: "generic", width: 6, height: 4) {
+		tileAttribute ("device.alarm", key: "PRIMARY_CONTROL") {
+			attributeState "off", label:'off', action:'alarm.siren', icon:"st.alarm.alarm.alarm", backgroundColor:"#ffffff"
+			attributeState "both", label:'alarm!', action:'alarm.off', icon:"st.alarm.alarm.alarm", backgroundColor:"#e86d13"
+		}
+	}
+	 
+	 valueTile("temperature", "device.temperature", inactiveLabel: false, width: 2, height: 2) {
+		 state "temperature", label:'${currentValue}°',
+			 backgroundColors:[
+				 [value: 32, color: "#153591"],
+				 [value: 44, color: "#1e9cbb"],
+				 [value: 59, color: "#90d2a7"],
+				 [value: 74, color: "#44b621"],
+				 [value: 84, color: "#f1d801"],
+				 [value: 92, color: "#d04e00"],
+				 [value: 98, color: "#bc2323"]
+			 ]
+	 }
+		
+	valueTile("humidity", "device.humidity", inactiveLabel: false, width: 2, height: 2) {
+		 state "humidity", label:'${currentValue}% humidity', unit:""
+	 }
+	
+	valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+		 state "battery", label:'${currentValue}% battery', unit:""
+	 }
+	
+	standardTile("refresh", "command.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+		state "default", label:'', action:"refresh.refresh", icon:"st.secondary.refresh"
+	}
+
+	valueTile("tamper", "device.tamper", height: 2, width: 2, decoration: "flat") {
+		state "clear", label: 'tamper clear', backgroundColor: "#ffffff"
+		state "detected", label: 'tampered', backgroundColor: "#ff0000"
+	}
+   
+	main "alarm"
+	details(["alarm", "humidity", "battery", "temperature", "tamper", "refresh"])
+	
+	}
+}
+
+def installed() {
+	initialize()
```

Comment:
can we make this a `runIn` here and in `configure()` so we don't call it twice?

Reply:
I don't get it. It is once invoke in installed().

Reply:
`configure()` also gets called on installation (for devices with the "Configuration" capability) after `installed()`